modules/installation-rhv-specifying-rhcos-image-settings.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installation-rhv-specifying-rhcos-image-settings_{context}"]
+= Specifying the {op-system} image settings
```

Comment:
I'd consider combining this module and "Downloading the Ansible playbooks."

Reply:
I agree. However, it looks like we're going to completely revise the process for getting those playbooks to avoid using github. So for now, let's keep it separate. I'll mark it TBD for reconsideration in https://issues.redhat.com/browse/OCPRHV-414